packages/tooltips/src/elements/Tooltip.tsx

```diff
@@ -141,20 +177,12 @@ const Tooltip = ({
 };
 
 Tooltip.propTypes = {
-  /** Appends the tooltip to the body element */
-  appendToBody: PropTypes.bool,
-  arrow: PropTypes.bool,
-  children: PropTypes.node,
-  /** Milliseconds of delay before open/close of tooltip is initiated */
+  appendToNode: PropTypes.any,
+  hasArrow: PropTypes.bool,
   delayMilliseconds: PropTypes.number,
-  /** Whether Popper.js should update based on DOM resize events */
   eventsEnabled: PropTypes.bool,
   id: PropTypes.string,
-  trigger: PropTypes.node,
-  /**
-   * These placements differ from the default naming of Popper.JS placements to help
-   * assist with RTL layouts.
-   **/
+  content: PropTypes.any.isRequired,
```

Comment:
What do you think about using`PropTypes.node.isRequired` for the `content` prop?